test/Conversion/ExportVerilog/sv-dialect.mlir

```diff
@@ -445,6 +445,16 @@ hw.module @struct_field_inout2(%a: !hw.inout<struct<b: !hw.struct<c: i1>>>) {
   sv.assign %1, %true : i1
 }
 
+// CHECK-LABEL: module AggregateConstantXZ(
+hw.module @AggregateConstantXZ() -> (res1: !hw.struct<foo: i2, bar: !hw.array<3xi4>>,
+                                     res2: !hw.struct<foo: i2, bar: !hw.array<3xi4>>) {
+  %0 = sv.constantX : !hw.struct<foo: i2, bar: !hw.array<3xi4>>
+  %1 = sv.constantZ : !hw.struct<foo: i2, bar: !hw.array<3xi4>>
+  // CHECK: assign res1 = 14'bx
+  // CHECK: assign res2 = 14'bz
```

Comment:
Wow, is this valid verilog syntax?  I guess it is because everything implicitly converts?

If so, this is great!

Reply:
Yes, I think it's valid:) I confirmed in EDA playground and verilator.